src/pages/NewGroupPage.js

```diff
@@ -155,6 +150,12 @@ class NewGroupPage extends Component {
      */
     createGroup() {
         const userLogins = _.map(this.state.selectedOptions, option => option.login);
+        if (userLogins.length < 1) {
+            return;
+        }
```

Comment:
Yeah this is fine for now IMO. 
I think ideally we would want to disable the Button until there are at least one selected option.

cc @shawnborton

Reply:
I agree with that. I don't know that we have disabled styles handy, but we could make them.

Reply:
We should have one for the submit button IIRC